kebechet/managers/update/update.py

```diff
@@ -922,6 +915,26 @@ def run(self, labels: list = []) -> Optional[dict]:
                             )
                         result = {}
                 results[self.runtime_environment] = result
-            close_manual_update_issue()
+
+            if overlays_dir:
+                issue = self.get_issue_by_title(_ISSUE_MANUAL_UPDATE)
+                if issue:
```

Comment:
These should be the outer conditional that way the `if issue` is checked for both conditional branches